src/core_plugins/kibana/public/doc/index.html

```diff
@@ -1,47 +1,75 @@
 <div ng-controller="doc" class="app-container">
+  <div class="kuiLocalNav">
```

Comment:
BTW I totally agree with you about consistency... using the top header consistently for every view is a good rule of thumb. But there are always exceptions to every rule, and I think we have some good reasons to make this situation an exception.